### ISSUE TITLE
 chef cookbook external project

### ISSUE THREAD

**Issue by yanick** [PerlDancer/Dancer2#232]
melhezik contributed a chef cookbook for Dancer 1 in PerlDancer/Dancer#914  We should see if it also applies to Dancer 2 and, if so, add it to the D2 docs as well.

**Comment by xsawyerx** [comment]
Not sure how valuable this is anymore. Maybe a Dockerfile would be better nowadays?

**Comment by cromedome** [comment]
Recent Dancer2 releases have a Dockerfile in the default app skeleton. Woo! :)

**Comment by yanick** [comment]
Yeah, we can totally kill the ticket. If one ever feel inspired to add a chef / ansible / docket recipe to the docs, a PR will appear anyway. :-)

**Comment by yanick** [comment]
> Recent Dancer2 releases have a Dockerfile in the default app skeleton. Woo! :)

Not only it appeared in a PR, but the PR was already there, was merged, and is already deployed.

MAGIC!!!
